src/routes/(app)/documents/+page.server.ts

```diff
@@ -107,6 +107,7 @@ export const actions = {
 
     form.delete("documents");
 
+    // @ts-ignore
     const update: Partial<Document> = Object.fromEntries(form);
```

Comment:
Instead of ignoring, consider casting the type: 
```suggestion
    const update: Partial<Document> = Object.fromEntries(form) as Partial<Document>;
```

Reply:
The error is coming from calling `Object.entries` on `form`. I'm not sure why, but it's not actually related to this PR, except that I needed to silence that warning for now.